tests/foreman/ui/test_hw_model.py

```diff
@@ -56,14 +54,15 @@ def test_create_negative_with_blank_name(self, name):
 
         """
         with Session(self.browser) as session:
-            make_hw_model(session, name=name)
-            error = session.nav.wait_until_element(
-                common_locators['name_haserror'])
-            self.assertIsNotNone(error)
+            for name in '', '   ':
```

Comment:
Yes this was an old PR, I figured that new strategy a little later.  I can fix this one.